### DISCUSSION TITLE
Akraba ilişkileri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#297] [Zeka Sorusu] [answered]
Hasan, Onur'un halasının oğludur. Burcu, Onur'un karısı ve amcasının kızıdır. Zeynep ise Hasan'ın teyzesinin kızıdır. Bu durumda Burcu, Zeynep'in nesi olur?

**Comment by adaakoluk** [comment, marked as answer]
![20230620_124352](https://github.com/sonsuzus/sonsuzus/assets/137164002/fb127d4d-8bd4-494d-a393-b246e3111566)
Burcu, Zeynep'in kuzenidir.
